### ISSUE TITLE
Multiple C++ Nuget Packages in one project

### ISSUE THREAD

**Issue by sbehnke** [coapp/coapp.powershell#2]
I'm attempting to add multiple native NuGet packages into one projects and something seems to not be working right.  I've added cpprestsdk, cpprestsdk.redist first and everything is fine.  I can #include <http_client.h> and compile.  Then I added bzip2 and bzip2.redist but when I attempt to #include <bzlib.h> Visual Studio tells me that it cannot find the file.  I also get a LNK2019 error: unresolved external symbol __imp___CrtDbgReportW references in public: static void __cdecl pplx::details::_Task_impl_base::_AsyncInit<...>.  If I remove the #include for bzlib.h this goes away and the project compiles again without issue.

I'm using Windows 7 SP1 x64 with Visual Studio 2012 Update 2 and NuGet 2.5.40416.9020.

**Comment by fearthecowboy** [comment]
Hmm. 

Just for giggles, can you switch to a Release build and compile? 

I know that we've found some cases where some libraries conflict on which version of the CRT it's trying to link with.

**Comment by sbehnke** [comment]
Yes, that seemed to build even with header files included from both projects.  So it appears it is isolated to the Debug build.

**Comment by fearthecowboy** [comment]
Indeed. 

Ok, I think I can understand the problem -- one library probably only supports Release, and therefore is probably implicity setting the CRT to release. 

_sigh_ 

then the other library has imports for functions that don't exist in Release, just debug.

This is gonna get harder to fix before it gets better I think.

**Comment by sbehnke** [comment]
Sorry, but thanks for all the hard work thus far!
